db/tests/test_backuptool.py

```diff
@@ -0,0 +1,47 @@
+"""Test for db/backuptool"""
+from constants import DATA_FOLDER
+import os
+import unittest
+
+import db.backuptool as bt
+
+TEST_FOLDER = 'test'
+TEST_TAR = 'test.tar.gz'
+TEST_FILE_1 = 'test.txt'
+TEST_FILE_2 = TEST_FOLDER + os.sep + 'test2.txt'
+
```

Comment:
PEP8-nitpicking: Please add an extra blank line here, before the class definition.